_posts/2017-05-26-deploying-create-react-app-to-s3.markdown

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title:  "Deploying create-react-app via S3"
```

Comment:
Needs a different title. It gets real specific about deploying to an Elixir app, and I think a more focused title could capitalize on some of the trendiness of Elixir/Phoenix.

Maybe something like, "Integrating Create React App with Elixir + Phoenix", or "Create React App, Elixir + Phoenix and Amazon S3: Awesome Sauce"